library/postfixadmin

```diff
@@ -1,18 +1,18 @@
-# This file is generated via https://github.com/postfixadmin/docker/blob/a6b5fe2423a7e1a805d1fcff910049768c638cff/generate-stackbrew-library.sh
+# This file is generated via https://github.com/postfixadmin/docker/blob/8205c9e5eff6190c76edf6e59f8c28eb2b0306b2/generate-stackbrew-library.sh
 Maintainers: David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)
 GitRepo: https://github.com/postfixadmin/docker.git
 
-Tags: 3.2-apache, 3-apache, apache, 3.2, 3, latest
-Architectures: amd64, arm64v8, i386
+Tags: 3.2.2-apache, 3.2-apache, 3.2.2, 3.2
```

Comment:
Did you mean to remove the `3` and `latest` tags? (and `fpm` and `fpm-alpine` below)

Reply:
Oops, thanks for the hint!